### ISSUE TITLE
Grid DropdownList header filter does not announce in NVDA 

### ISSUE THREAD

**Issue by kietresolve** [telerik/kendo-angular#1766]
## I'm submitting a...

* Bug report 

## Current behavior

Currently in Kendo Grid [filterable]="true" the filter dropdown value changes or selection are not announced in NVDA screen reader. I have reviewed the arias used which seem to be correct but still it seems to be invisible to NVDA.

But the Kendo JQuery version works which can be found here https://demos.telerik.com/kendo-ui/grid/filter-row

## Expected behavior

For the filter dropdown values to be announced in screen reader like NVDA... like its Kendo Jquery counterpart.  https://demos.telerik.com/kendo-ui/grid/filter-row

## Minimal reproduction of the problem with instructions

https://stackblitz.com/edit/angular-1mqr6q?file=main.ts

if link above does not work use this alternative https://www.telerik.com/kendo-angular-ui/components/grid/filtering/

Please have NVDA running and select on the filter button in the header and then use keyboard to navigate the dropdown values, then you will see that the values in the dropdown are not announced when selection is changed or selected.

**Comment by danielkaradachki** [comment]
Possibly related to: https://github.com/telerik/kendo-angular/issues/1070

**Comment by danielkaradachki** [comment]
fixed in version 3.1.0 of the dropdowns package.
